### PR TITLE
Feature/110 order routing

### LINKED ISSUE #110: Order detail route should be by order nonce
Instead of `nft-detail/:token-id/buy` the route should be `nft-detail/:token-id/:order-nonce` because it's possible to have multiple orders for one token.

### PR DESCRIPTION
**Pull request by piersss**
# Description

Fixes #110 

Route of an order is now: /order/:account/:orderNonce

# Steps to test the change

1. Make an order
2. Click "View Listing"

# Checklist:

- [x] I have read and understood [CONTRIBUTING.MD](https://github.com/airswap/airswap-marketplace/blob/main/CONTRIBUTING.md)
- [x] I have included the ticket number in my PR
- [x] I have performed a self-review of my own code
- [x] I have commented the parts where my code is hard to understand

### CODE REVIEW

**Comment by personnamedmike** [comment]
`Route of an order is now: /order/:account/:orderNonce`

Visiting this route returns an error. The route order I see is: `/order-detail/:account/:orderNonce`

File: src/widgets/BuyNftWidget/BuyNftWidget.scss [hunk 1]
```diff
@@ -1,8 +1,6 @@
 @import "src/styles";
 
-$component: buy-nft-widget;
-
-.#{$component} {
+.buy-nft-widget {
```
Comment (personnamedmike) [inline src/widgets/BuyNftWidget/BuyNftWidget.scss:3]: where exactly is this class being used?

**Review by personnamedmike — APPROVED** [review]

File: src/widgets/BuyNftWidget/subcomponents/BuyActionButtons/BuyActionButtons.tsx [hunk 2]
```diff
@@ -101,7 +101,7 @@ const BuyActionButtons: FC<ActionButtonsProps> = ({
     if (state === BuyNftState.success) {
       return (
         <NavLink
-          to={`/${AppRoutes.profile}/${account}`}
+          to={routes.profile(account)}
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/widgets/BuyNftWidget/subcomponents/BuyActionButtons/BuyActionButtons.tsx:104]: Yeah `account` should be in here.

File: src/widgets/BuyNftWidget/BuyNftWidget.scss (hunk 1 above)
Comment (piersss) [reply src/widgets/BuyNftWidget/BuyNftWidget.scss:3]: In `ConnectedBuyNftWidget`

**Pull request merged**